app/assets/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,37 @@
+.centered {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
how does this look on a landscape phone? I think this should be `min-height` instead, to make sure that we never make it shorter than it needs to be.